### DISCUSSION TITLE
Would a 'Building ZFSBootMenu' quick start guide be handy?

### DISCUSSION THREAD

**Discussion by jrtoddy** [zbm-dev/zfsbootmenu#270] [General]
Hello everyone,

While figuring things out working on #268, I took some notes and wrangled them into a little guide. Let me know what you think, could it be useful for newcomers? Also, please check for veracity. :sweat_smile:

I haven't dumped the raw markdown below to avoid polluting future searches with misinformation and likely typos but, oddly, the Github won't let me attach a .md, .MD or .markdown file directly so it's a ZIP instead.

[BUILD.md.gz](https://github.com/zbm-dev/zfsbootmenu/files/8190462/BUILD.md.gz)

**Comment by zdykstra** [comment]
A build guide would be a fantastic addition! I have a few changes I'd like to suggest. If you'd like to throw `BUILD.md` into a PR, we can use that as a review tool. Thank you!
